app/services/audit_log_service.rb

```diff
@@ -0,0 +1,7 @@
+class AuditLogService
+  include Callable
+
+  def self.call(message)
+    Rails.logger.info "[AUDIT]: #{message}"
+  end
+end
```

Comment:
No new line at the end of the file!

Reply:
It's a good practice. I don't know why codeclimate doesn't complain about it. Fixed.